electron/legendary_utils/library.ts

```diff
@@ -20,46 +19,33 @@ import { Game, InstalledInfo, KeyImage } from '../types'
 const execAsync = promisify(exec)
 const statAsync = promisify(stat)
 
-export async function getLegendaryGames() {
-  const { stdout, stderr } = await execAsync(
-    `${legendaryBin} list-games --json`
-  )
-  if (stdout) {
-    const results = JSON.parse(stdout)
-    // NEED DOUBLE CHECK IF SOMETHINGS INSIDE IS NEEDED OR ADD MORE STUFF IT REDUCE SIZE OF THE FINAL FILE
-    // There is maybe a better way to do that tried with filter but still not work
-    results.map((res: any) => {
-      delete res.asset_info
-      delete res.metadata.creationDate
-      delete res.metadata.developerId
-      delete res.metadata.dlcItemList
-      delete res.metadata.endOfSupport
-      delete res.metadata.entitlementName
-      delete res.metadata.entitlementType
-      delete res.metadata.lastModifiedDate
-      delete res.metadata.releaseInfo
-      res.metadata.keyImages.map((k: any) => {
-        delete k.height
-        delete k.md5
-        delete k.size
-        delete k.uploadedDate
-        delete k.width
-      })
-    })
-    const json = JSON.stringify(results)
-    return writeFileSync(`${heroicFolder}library.json`, json)
-  } else if (stderr) {
-    // return a file to avoid infinite loop
-    return writeFileSync(
-      `${heroicFolder}library.json`,
-      '{"result":"noGamesFound"}'
-    )
-  } else {
-    // return a file to avoid infinite loop
-    return writeFileSync(
-      `${heroicFolder}library.json`,
-      '{"result":"noGamesFound"}'
-    )
+export async function getLegendaryGames(): Promise<void> {
+  const {stdout, stderr} = await execAsync(`${legendaryBin} list-games --json`, {maxBuffer: Infinity})
+  const stdoutjson = JSON.parse(stdout.toString())
+  const result = []
```

Comment:
Even though this works, would be more cleaner here if you could use an Array.map function:
`const result = stdoutjson.map(game => {"app_name": game.app_name, ...})`
I think the above will work.
dont forget to set the types fro stdoutjson. Probably will be and array of Game, so `Array<Game>`